### PR TITLE
Tail command

### LINKED ISSUE #463: Writing Documentation for "tail"
jcambro@umich.edu

### PR DESCRIPTION
**Pull request by jcambro**
# Advanced HW 5 submission

Closes #463 

Name: John Ambrose
Uniqname: jcambro

### CODE REVIEW

File: _commands/basics/tail.md [hunk 1]
```diff
@@ -3,22 +3,42 @@
...
 -------
-TODO: Add documentation for this command by submitting a pull request.
+`tail` is used to read the final few lines of text from a document. The default length is 10. 
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ tail example.txt
+$ tail example1.txt example2.txt
```
Comment (AmritHariharan) [inline _commands/basics/tail.md]: what does tail do when given two commands? Add this to the useful options and explain what it does.

File: _commands/basics/tail.md [hunk 2]
```diff
@@ -3,22 +3,42 @@
...
+ * The `c` flag allows you to only print the last k amount of bytes.
+
+#### `tail -f`
+
+~~~ bash
+$ tail -f example.txt
+~~~
 
 ##### Break it down
+ * The `f` flag follows the file and outputs any new text added to the end of the file.
```
Comment (AmritHariharan) [inline _commands/basics/tail.md]: I'm not sure what you mean here, add sample output to show what it actually does.

**Comment by jcambro** [comment]
I added another commit. Is there anything else I need to merge again or is it just that one commit on my forked branch?

**Comment by AmritHariharan** [comment]
You have been awarded credit for the HW + 2 points extra credit for submitting the initial PR early.

**Pull request closed without merging**